### PR TITLE
Auto Help Str

### PR DESCRIPTION
**Pull request by tomstitt**
- automatically update help str on new streams
- `make style`
- add util function to check if the device is a touch device

### CODE REVIEW

**Review by tzanio — APPROVED** [review]
Works great, thanks!

**Comment by tzanio** [comment]
Sorry about forgetting to run `make style`!

**Comment by tomstitt** [comment]
> Sorry about forgetting to run `make style`!

No worries! Prettier ran into an error parsing `<button type="button" onclick="glv.sendKey('\"')">"</button>` so maybe you did and just missed the error?

File: README.md [hunk 1]
```diff
@@ -74,6 +74,25 @@ Example usages:
 NOTE: Emscripten handles SDL2 and GLEW but if you have another installation in your path the link
 might fail.
 
+## Serving to a device on your local network
```
Comment (tzanio) [inline README.md:77]: Thanks for adding this!

**Pull request merged**